apps/dokuha-roadmap-frontend/biome.json

```
@@ -1,30 +1,71 @@
 {
 	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
-	"vcs": {
-		"enabled": false,
-		"clientKind": "git",
-		"useIgnoreFile": false
-	},
-	"files": {
-		"ignoreUnknown": false,
-		"ignore": []
-	},
+	"vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": false },
+	"files": { "ignoreUnknown": false, "ignore": [] },
```

biome が .md とかも弾いてるクサイ
```suggestion
	"files": { "ignoreUnknown": true, "ignore": [] },
```